docs/area-owners.md

```diff
@@ -89,9 +89,9 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
 | area-System.DirectoryServices                  | @ericstj             | @dotnet/area-system-directoryservices               | Consultants: @BRDPM @grubioe @jay98014                                                                                                                                                                                                                                                    |
 | area-System.Drawing                            | @ericstj             | @dotnet/area-system-drawing                         | Excluded:<ul><li>System.Drawing.Common -> winforms</li></ul>                                                                                                                                                                                                                              |
 | area-System.Dynamic.Runtime                    | @jaredpar            | @cston @333fred                                     | Archived component - limited churn/contributions (see [#27790](https://github.com/dotnet/runtime/issues/27790))                                                                                                                                                                           |
-| area-System.Formats.Asn1                       | @jeffhandley         | @dotnet/area-system-formats-asn1                    |                                                                                                                                                                                                                                                                                           |
-| area-System.Formats.Cbor                       | @jeffhandley         | @dotnet/area-system-formats-cbor                    |                                                                                                                                                                                                                                                                                           |
-| area-System.Formats.Tar                        | @jeffhandley         | @dotnet/area-system-formats-tar                     | Consultants: @bartonjs @jozkee                                                                                                                                                                                                                                                            |
+| area-System.Formats.Asn1                       | @jeffhandley         | @dotnet/area-system-formats-asn1                    | Consultants: @bartonjs                                                                                                                                                                                                                                                                    |
```

Comment:
If you want, feel free to mark me available for consulting for `area-System.Formats.Asn1` and `area-System.Formats.Cbor`.